src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -96,20 +96,85 @@ void createStationWithDuplicateName() {
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
-     * Then 2개의 지하철역을 응답 받는다
+     * Then 지하철역 목록을 응답 받는다
+     * Then 생성한 지하철역 2개가 응답받은 지하철역 목록에 존재한다
      */
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // given
+        String[] createStations = new String[] {"강남역", "교대역"};
+        for (String createStation : createStations) {
+            createStationForTest(createStation);
+        }
+
+        // when
+        ExtractableResponse<Response> response =
+                RestAssured.given().log().all()
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().get("/stations")
+                        .then().log().all()
+                        .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+        // then
+        List<String> stationsNames = response.jsonPath().getList("name", String.class);
+        for (String createStation : createStations) {
+            assertThat(stationsNames.contains(createStation)).isTrue();
+        }
     }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
+     * Then 지하철역이 삭제된다
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        // given
+        String stationId = createStationForTest("강남역");
+
+        // when
+        ExtractableResponse<Response> response =
+                RestAssured.given().log().all()
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().delete("/stations/"+stationId)
+                        .then().log().all()
+                        .extract();
+
+        // then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+
+        // then
+        List<String> stationNamesAfterDelete =
+                RestAssured.given().log().all()
+                        .when().get("/stations")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
```

Comment:
```suggestion
        List<String> 지하철_이름_목록 = 지하철_이름목록_조회();
```

`stationNamesAfterDelete` 제가 생각하기에는 다소 디테일하고 많은 의미를 가진 네이밍인것 같아요.
단순하게, `지하철_이름_목록` 으로해도 좋을것 같다는 생각이 들기도하네요.

꼭 변경해야되는 부분은 아니며, 관점 공유라고 봐주시면 감사하겠습니다!